packages/next/src/server/image-optimizer.ts

```diff
@@ -734,14 +798,14 @@ export function sendResponse(
   url: string,
   extension: string,
   buffer: Buffer,
+  etag: string,
   isStatic: boolean,
   xCache: XCacheHeader,
   imagesConfig: ImageConfigComplete,
   maxAge: number,
   isDev: boolean
 ) {
   const contentType = getContentType(extension)
-  const etag = getHash([buffer])
```

Comment:
If we are storing the etag already, shouldn't we use that instead of recalculating the `etag`? This can be reverted to use `getImageTag`.